### PR TITLE
Navigation between multiple ListView

### PR DESCRIPTION
**Pull request by asabotovich**
## PR includes

- [x] Bug Fix

## Related issues

Resolve: #752 

## QA Instructions, Screenshots, Recordings

https://github.com/taskany-inc/bricks/assets/9020793/6e923a75-3f9e-4c53-8c36-087f0a407542

### CODE REVIEW

File: src/hooks/useKeyPress.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { useState, useEffect } from 'react';
 
-export const useKeyPress = (targetKey: string) => {
+export const useKeyPress = (targetKey: string, enabled = true) => {
```
Comment (awinogradov) [inline src/hooks/useKeyPress.tsx:3]: Use if statement in place of usage instead of this argument

**Review by awinogradov — CHANGES_REQUESTED** [review]

**Review by awinogradov — APPROVED** [review]

**Review by tsumo — APPROVED** [review]

**Pull request merged**